src/mm/dbcsr_mm.F

```diff
@@ -921,12 +920,6 @@ SUBROUTINE dbcsr_multiply_generic(transa, transb, &
                                      filter_eps=filter_eps, &
                                      flop=my_flop, keep_product_data=keep_product_data)
          END IF
-#else
```

Comment:
@alazzaro , are you sure this change removing the else part is ok? Is it ok for some jobs to crash because this code path only supports real_8 data?

Reply:
We still have the else... it was there even before:
```
         IF (data_type .NE. dbcsr_type_real_8 .OR. (.NOT. dbcsr_cfg%use_acc_g2g%val)) THEN
            ! If G2G is enabled, norms have to be calculated on the GPU.
            ! Since the norms kernel expects only real_8 type data, we
            ! avoid using G2G for all other data types
            CALL multiply_cannon(m2s_left, m2s_right, product_matrix, &
                                 retain_sparsity=retain_sparsity, &
                                 filter_eps=filter_eps, &
                                 flop=my_flop, keep_product_data=keep_product_data)
         ELSE
            CALL multiply_cannon_g2g(m2s_left, m2s_right, product_matrix, &
                                     retain_sparsity=retain_sparsity, &
                                     filter_eps=filter_eps, &
                                     flop=my_flop, keep_product_data=keep_product_data)
         END IF
```
Am I right?

Reply:
Yes you are absolutely right. I didn't read the changed file carefully. Thanks @alazzaro!